### DISCUSSION TITLE
Cloudlog general querie

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#1918] [Q&A]
Hello

I’m considering moving totally to Linux but don’t want to have any of the databases local. I’m looking for something that I can simply have WSJT-X upload to LOTW (at least) via the browser without relying on complex local software. 

The logbook should reside in the cloud. 

Is Cloudlog something for me?

Thank you guys

Scott, W1AL

**Comment by phl0** [comment]
I am logging into Cloudlog. FT8 QSOs are uploaded from WSJT-X improved to Cloudlog automatically. Cloudlog then syncs with LotW every hour.
